### ISSUE TITLE
No progress shown until mouse hover over progress window

### ISSUE THREAD

**Issue by riksoft** [PortableApps/Toucan#37]
After launching a backup, the progress window appears, but unless you move the mouse over it, it shows nothing.

This way the software seems busy gathering data before starting the backup and the user simply waits... forever because the backup is already finished but you won't notice anything unless you move the mouse over the progress window.
V. 3.1.8

**Comment by riksoft** [comment]
Looking at the code directly from the pages of github, I'd say the problem is 99% in the file
backup/backupprocess.cpp
where the BackupProcess::HasInput() doesn't return true... probably because IsInputAvailable() doesn't trigger... probably because the input stream is wrong (maybe is in the error stream?), and moving the mouse is actually the only thing that triggers IsInputAvailable().

**Comment by tedd-wq** [comment]
I'm also (still) getting this problem. I had issue #32

**Comment by alex-ter** [comment]
Closing as a duplicate of #32
